next-packages/serialize/src/index.js

```diff
@@ -72,8 +85,8 @@ export function handleInterpolation(
   }
   if (interpolation.__emotion_styles !== undefined) {
     if (
-      interpolation.toString() === 'NO_COMPONENT_SELECTOR' &&
-      process.env.NODE_ENV !== 'production'
+      process.env.NODE_ENV !== 'production' &&
```

Comment:
is this change purely because of the code style or does it impact something?

Reply:
Without this change, uglify can't remove the `interpolation.toString()` call since it doesn't know if it has side effects. We could probably add a pure comment but this is an easy change and will work regardless of the minifier.